com.unity.ml-agents/CHANGELOG.md

```diff
@@ -13,8 +13,11 @@ and this project adheres to
 ### Minor Changes
 
 ### Bug Fixes
-`mlagents-learn` will now raise an error immediately if `--num-envs` is greater than 1 without setting the `--env`
```

Comment:
This got cherry-picked to the release branch, so it should get merged back into the section for 1.2.0